app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,32 @@
+class SessionsController < ApplicationController
+  def new
+    if current_user
+      redirect_to home_path 
+    else
+      @user = User.new
+    end
+  end
+
+  def create
+    user = User.where(email: user_params[:email]).first
```

Comment:
You should use `.find_by` instead of `where(...).first`. It's more efficient and Rails 4 preferred way for searching a single record.